lib/signs/utilities/audio.ex

```diff
@@ -238,18 +251,6 @@ defmodule Signs.Utilities.Audio do
     end)
   end
 
-  @spec sort_audio([Content.Audio.t()]) :: [Content.Audio.t()]
-  defp sort_audio(audios) do
-    Enum.sort_by(audios, fn audio ->
-      case audio do
-        %Content.Audio.TrainIsBoarding{} -> 1
-        %Content.Audio.TrainIsArriving{} -> 2
-        %Content.Audio.Approaching{} -> 3
-        _ -> 4
-      end
-    end)
-  end
-
```

Comment:
Even in the old code, an arriving message would suppress the readout of a normal prediction, so that particular situation wouldn't happen. However, an important point is that this sorting only happens within the context of a simultaneous update. If the statuses came in 1 second apart, the first one to change would get read first, regardless. That's why I think this isn't something we should worry about.

Reply:
Ah okay, I see what you mean now